packages/components/src/highlight-cards/highlight-card.tsx

```diff
@@ -74,7 +74,8 @@ export default function HighlightCard( {
 							{ difference > 0 && <Icon size={ 18 } icon={ arrowUp } /> }
 						</span>
 						<span className="highlight-card-difference-absolute-value">
-							{ Math.abs( difference ) }
+							{ difference <= 9999 && formattedNumber( Math.abs( difference ) ) }
+							{ difference > 9999 && <ShortenedNumber value={ Math.abs( difference ) } /> }
```

Comment:
```suggestion
							{ Math.abs( difference ) <= 9999 && formattedNumber( Math.abs( difference ) ) }
							{ Math.abs( difference ) > 9999 && (
								<ShortenedNumber value={ Math.abs( difference ) } />
							) }
```

We need to consider negative differences when calculating the magnitude cutoff. For example:

Without this change:
<img width="1259" alt="image" src="https://user-images.githubusercontent.com/4044428/217639772-a56136d5-79ba-48d9-9ce1-5c926d50fa0a.png">

With this change:
<img width="1251" alt="image" src="https://user-images.githubusercontent.com/4044428/217640317-839e6cc4-360e-4232-be45-07c437c76025.png">

Reply:
Addressed in c46afb5.